_posts/2021-10-06-what-is-seraphis.md

```diff
@@ -0,0 +1,57 @@
+        - 'Membership proof delegation' is possible without leaking users' private keys. This potentially allows the following:
+            - Transaction chaining (required for full-featured atomic swaps).
+            - Offload membership proof construction to a third party, or a view-only wallet (possibly useful for spend-only hardware wallets where it is expensive/difficult to implement complex algorithms).
+            - Ignore 10-block lock time when transacting with a **trusted party** (i.e. allow them to make your tx's membership proofs and submit the tx to the network on your behalf).
+        - Flexible multi-tier address schemes. The following features can be designed into a Seraphis user address scheme (there are at least 7 design variations, half of which require 3-key addresses [a 50% increase in address length over current addresses]):
+            - View-only wallet that can see spent outputs.
+                - Especially helpful for multisig and hardware wallets, since in RingCT and CryptoNote it is necessary to constantly 'export key images'.
+            - View-only wallet that can see received outputs but not amounts (and optionally also see spent outputs but not amounts).
+                - Useful for delegated chain scanning, where a third party or insecure machine scans for outputs, then sends them to a more secure 2nd tier view-only wallet (or spend wallet) that can read the amounts.
+            - View-only wallet that can detect the Janus attack.
+        - Modular core design. This will hopefully make it easier to upgrade different pieces of a Seraphis implementation as innovations appear (or maybe we will be stuck for the next 20 years... who knows?).
+    - Cons
+        - If a cryptocurrency uses RingCT, it is more work to upgrade to Seraphis than Triptych.
+        - Not compatible with CryptoNote user addresses. If Seraphis is used to upgrade a cryptocurrency that has CryptoNote-style addresses, then users would have to replace all their public CryptoNote-style addresses with Seraphis-style addresses (they would not need new private keys or wallets).
+
+
+### What about Lelantus-Spark?
+
+[Lelantus-Spark](https://eprint.iacr.org/2021/1173) is a transaction protocol (but not an abstraction - like how Triptych is not an abstraction) very similar to Seraphis, with the same pros/cons relative to Triptych. It was developed independently from Seraphis, for the most part.
+
+Since they were developed at the same time, the question 'why Seraphis and not Lelantus-Spark?' can just as easily be stated 'why Lelantus-Spark and not Seraphis?' Practically speaking, the authors of Lelantus-Spark have focused their attention on design details appropriate for the Firo cryptocurrency, while my design recommendations for Seraphis (in the paper) have focused on details appropriate for a cryptocurrency that currently uses standard RingCT (like Monero or MobileCoin).
```

Comment:
> Before anyone whines about my mention of MobileCoin, you should keep in mind I wrote the first draft of the Seraphis paper while an employee at MobileCoin.

In that case i would either remove the mention of MobileCoin entirely or i would add a clarification explaining why it's mentioned. Otherwise it looks out of place (without context there is no reason to mention MobileCoin an not other, more prominent, coins that use ringct)